client/me/purchases/manage-purchase/index.jsx

```diff
@@ -681,6 +685,13 @@ const ManagePurchase = React.createClass( {
 					{ contactSupportToRenewMessage }
 				</Card>
 
+				<PurchasePlanDetails
```

Comment:
Can we completely avoid rendering this one if we know it's not a Jetpack site? `isJetpackSite` could come in handy.